src/lib/core/fiber.c

```diff
@@ -1827,3 +1827,12 @@ int fiber_stat(fiber_stat_cb cb, void *cb_ctx)
 	}
 	return 0;
 }
+
+struct lua_State *
+fiber_lua_state_internal(struct fiber *fiber)
+{
+	if (fiber == NULL)
+		return NULL;
```

Comment:
I would assert instead.

Reply:
I think that check on `NULL` would be preferable because this function will be used in user code (on `RELEASE` builds) and `assert` only works on `DEBUG` builds.
Why do you think that `assert` is preferable?

Reply:
Because passing NULL as a pointer to the fiber is definitely incorrect action, it is better to fail faster. We're doing the same for `box_tuple_ref()` and `box_tuple_unref()`, so it seems quite usual.

Reply:
ok